www/include/common/javascript/datepicker/localizedDatepicker.js

```diff
@@ -134,26 +135,29 @@ function turnOffEvents() {
     $(".timepicker").off('change');
 }
 
+
 function updateEndTime() {
+    var durationScale = { "d" : 86400, "h" : 3600, "m": 60, "s": 1 };
     var start = new Date($(".datepicker").first().val() + ' ' +  $(".timepicker").first().val());
     var end = new Date($(".datepicker").last().val() + ' ' +  $(".timepicker").last().val());
     if (start > end) {
         turnOffEvents();
         var e = new Date();
-        e.setTime(start.getTime() + 7200000); //microseconds
+        e.setTime(start.getTime() + $('#duration').val() * durationScale[$('#duration_scale').val()] * 1000); //milliseconds
         $(".datepicker").last().datepicker("setDate", e);
         $(".timepicker").last().timepicker("setTime", e.getHours() + ':' + e.getMinutes());
         turnOnEvents();
     }
 }
 
 function updateStartTime() {
+    var durationScale = { "d" : 86400, "h" : 3600, "m": 60, "s": 1 };
```

Comment:
The duration scale should be set by the back end, not the front

Reply:
https://github.com/centreon/centreon/pull/7606#discussion_r351916973